src/solvers/dgsem_tree/dg_parallel.jl

```diff
@@ -5,6 +5,16 @@
 @muladd begin
 #! format: noindent
 
+# Function to output the mpi rank for visualization
+function get_element_variables!(element_variables,
+                                mesh::Union{ParallelTreeMesh, ParallelP4estMesh,
+                                            ParallelT8codeMesh},
+                                dg, cache)
+    element_variables[:mpi_rank] = fill(mpi_rank(),
+                                        SVector{nelements(mesh, dg, cache), real(dg)})
```

Comment:
```suggestion
    element_variables[:mpi_rank] = fill(mpi_rank(), nelements(mesh, dg, cache))
```
I think `SVector` is not suitable for large data structures such at this. Also, you don't want to store the MPI rank as a real number but rather as an integer, don't you?

Reply:
> I think SVector is not suitable for large data structures such at this. 

If not an `SVector`, then what should I use?

Reply:
> > Also, you don't want to store the MPI rank as a real number but rather as an integer, don't you?
> 
> I tried that, but it did not work because the element variables are written as real numbers in the solution file.

Ah, got it.



> > I think SVector is not suitable for large data structures such at this.
> 
> If not an `SVector`, then what should I use?

Just a regular array. For example,
```suggestion
    element_variables[:mpi_rank] = ones(real(dg), nelements(mesh, dg, cache)) * mpi_rank()
```
should do the trick with the correct element type type.

Reply:
Thanks!